x/docs/internal/types/document_test.go

```diff
@@ -19,3 +19,38 @@ func TestDocument_Equals_NilValues(t *testing.T) {
 	}
 	assert.True(t, document.Equals(document))
 }
+
+func Test_validateUUID(t *testing.T) {
+	tests := []struct {
+		name    string
+		UUID    string
+		badUUID bool
+	}{
+		{
+			"empty string",
+			"",
+			true,
+		},
+		{
+			"a well-formed UUID",
+			"6ba7b810-9dad-11d1-80b4-00c04fd430c8",
+			false,
+		},
+		{
+			"a seemingly well-formed UUID, with the last character removed",
+			"6ba7b810-9dad-11d1-80b4-00c04fd430c",
+			true,
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Why do we need to call `t.Run`? Can't we just iterate over each one without calling `Run`?

Reply:
This is the standard testing template generated by [gotests](https://github.com/cweill/gotests).

I can refactor it to work without the `t.Run` call, although this pattern is useful to easily pin-point what test case failed (each `Test_XYZ` func has multiple subtests, defined by the `tests` slice).